### PR TITLE
Adding types to wix-ui-test-util

### PR DESCRIPTION
**Pull request by danielstro**

### CODE REVIEW

File: packages/wix-ui-test-utils/src/helpers.tsx [hunk 1]
```diff
@@ -19,20 +27,17 @@ export const sleep = (ms: number): Promise<void> => {
 };
 
 // HOC that makes underlying component "controlled"
-export function makeControlled(Component) {
+export function makeControlled(Component: React.SFC<React.AllHTMLAttributes<any>>) {
```
Comment (danielstro) [inline packages/wix-ui-test-utils/src/helpers.tsx]: This could be a mistake. Not sure

File: packages/wix-ui-test-utils/src/helpers.tsx [hunk 2]
```diff
@@ -7,10 +7,18 @@ export interface ControlledComponentState {
 
 export interface ControlledComponentProps {
   value?: string;
-  onChange?: (e: Event) => void;
+  onChange?: (e: React.ChangeEvent<any>) => void;
```
Comment (amiryonatan) [inline packages/wix-ui-test-utils/src/helpers.tsx]: React.ChangeEvent<HTMLElement>

File: packages/wix-ui-test-utils/src/protractor-helpers.ts [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-  browser.executeScript(el => {
+export const scrollToElement = (element: ElementArrayFinder) => {
+  browser.executeScript((el: HTMLElement)  => {
     const offset = el.offsetTop;
     window.scroll(0, offset);
   }, element.getWebElement());
 };
 
-export const waitForVisibilityOf = (elements, errorMsg, timeout = 10000) => {
+export const waitForVisibilityOf = (elements: Array<ElementFinder>, errorMsg?: string, timeout = 10000) => {
```
Comment (amiryonatan) [inline packages/wix-ui-test-utils/src/protractor-helpers.ts]: Array<ElementFinder> | ElementFinder

File: packages/wix-ui-test-utils/tsconfig.json [hunk 4]
```diff
@@ -1,19 +1,17 @@
...
-    "noUnusedLocals": true,
     "jsx": "react",
     "lib": ["dom", "es2016"],
-    "allowJs": true
+    "strict": true
   },
   "include": [
     "./src/**/*.tsx",
     "./src/**/*.ts",
-    "./src/**/*.js",
```
Comment (argshook) [inline packages/wix-ui-test-utils/tsconfig.json]: @amiryonatan @shlomitc wsr build is red because of this, i think

Comment (argshook) [reply packages/wix-ui-test-utils/tsconfig.json]: ignore me, i see you fixed here https://github.com/wix/wix-ui/commit/fde4bfdc40cc4af8025eed260dcc2857063ec14e

**Pull request merged**